tests/Set/Sulu24/Fixture/localization.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class SomeClass {
+    public function someMethod(\Sulu\Component\Localization\Localization $localization)
+    {
+        $localization->setXDefault('en');
+        $localization->getXDefault();
+        $localization->isXDefault();
+    }
+}
+
+?>
+---
+<?php
+
+class SomeClass {
+    public function someMethod(\Sulu\Component\Localization\Localization $localization)
```

Comment:
Does this class exist in "require-dev" dependencies of `composer.json` or local `/stubs` directory?

Without existing class, PHPStan cannot find it nor analyse it.